src/styles/block/calendar.scss

```diff
@@ -0,0 +1,58 @@
+.calendar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__block {
+    display: flex;
+
+    gap: $gap;
+    width: $width;
+    flex-wrap: wrap;
+
+    padding: 50px $calendar-padding 0;
+
+    @each $day, $number in $days-of-week {
+      &--start-day--#{$day} {
+        & :first-child {
+          margin-left: $calendar-day-width * $number + $gap * 6;
+        }
```

Comment:
Do you always need to multiply gap and 6? This is a problem that broke your calendar